locale/fi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,223 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Thank you so much for writing this file up 👏🏼

If I remember correctly, the locale file `djano.po` must be compiled (`django.mo`) so that django can use it. The practice that I have seen is that this compiled file is checked in the repository (for example, Wagtail is a famous CMS based on Django and [here are the Finnish locale files](https://github.com/wagtail/wagtail/tree/main/wagtail/admin/locale/fi/LC_MESSAGES)).

So, could you please check-in the compiled file to the repository?

In addition, could you please add `locale` folder to the `Allowed folders` in `.dockerignore` file so that the container would have access to the locale files too?

Reply:
Actually this is not true that the compiled translations should be in the repository. We actually have explicit gitignore-rule for this saying:

````
# Translations
*.mo
````

So I imagine this is the Django preferred way of doing this.
Also we should do the `compilemessages`-cmd call somewhere automatically, docker-entrypoint is typically one good place to do this, so I'll add it there now.

I'll also add the `locale`-folder to `Allowed folders` in `.dockerignore`